addon/components/frost-table-row.js

```diff
@@ -87,6 +88,15 @@ export default Component.extend({
 
   // == Lifecycle Hooks =======================================================
 
+  didRender () {
+    this.$().css({
+      'flex-grow': 1,
+      'flex-shrink': 0,
+      'flex-basis': `${this.setMinimumCellWidths('')}px`
```

Comment:
Does the `''` need to be passed?  Can't is just be`${this.setMinimumCellWidths()}px`?